jest.config.ts

```diff
@@ -6,7 +6,7 @@ const jestConfig: JestConfigWithTsJest = {
   setupFilesAfterEnv: ['jest-expect-message', 'jest-extended/all'],
   projects: [
     '<rootDir>/scripts/jest.config.ts',
-    '<rootDir>/services/!(vod-to-live)/**/jest.config.ts',
+    '<rootDir>/services/**/jest.config.ts',
     //'<rootDir>/libs/**/jest.config.ts',
```

Comment:
Please add a space after `//` and also run `yarn script` command from monorepo
root to do the same for other files. (this file seems to be ignored by `yarn
lint` script)